README.md

```diff
@@ -118,9 +118,9 @@ You'll need to install three pods, `MapboxNavigationUI.swift`, `MapboxNavigation
 Add the following lines to your Podfile:
 
 ```ruby
-pod 'MapboxDirections.swift', '~> 0.8'
-pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.0.4'
-pod 'MapboxNavigationUI.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :commit => 'a368a73a7575b296886ae53b7642216c167ca8e2'
+pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.1.0'
+pod 'MapboxNavigationUI.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.1.0'
+pod 'OSRMTextInstructions', :git 'https://github.com/Project-OSRM/osrm-text-instructions.swift.git' => , :tag => 'v0.1.0'
```

Comment:
With OSRMTextInstructions being versioned through CocoaPods.org, this line becomes unnecessary.